src/sql/account/migrations/20000000_old_schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE auth
```

Comment:
What are all these migration files in `sql/account/migrations/*`? AFAIK they are unused.

Reply:
I'm not sure. I thought that was already on master. Looks like it was created on the old PR from that I based this on

Reply:
fixed